README.md

```diff
@@ -5,124 +5,99 @@
 
 This repository houses the application code for [RemoteRetro.org](http://remoteretro.org), a free web app that allows distributed teams to conduct Agile retrospectives. It is written in Elixir/Phoenix/React/Redux, and is sponsored by [Stride Consulting](https://www.stride.build).
 
-## Table of Contents
+## Running with Docker
 
-1. [Dev Environment Setup (docker)](#dev-environment-setup-docker)
-1. [Dev Environment Setup (macOS)](#dev-environment-setup-macos)
-1. [Tests](#tests)
-1. [Code](#code)
-1. [Contributing](#contributing)
-1. [Code of Conduct](#code-of-conduct)
-1. [Acknowledgements](#acknowledgements)
-1. [License](#license)
```

Comment:
It's not a very big readme so I thought I'd delete this instead of maintaining it.